packages/fxa-auth-server/test/remote/db_tests.js

```diff
@@ -1288,6 +1288,40 @@ describe('remote db', function() {
       });
   });
 
+  it('db.securityEventsByUid', () => {
+    return db
+      .securityEvent({
+        ipAddr: '127.0.0.1',
+        name: 'account.create',
+        uid: account.uid,
+      })
+      .then(() => {
+        return db.securityEventsByUid({
+          uid: account.uid,
+        });
+      })
+      .then(events => {
+        assert.equal(events.length, 1);
+      });
+  });
+
+  it('db.deleteSecurityEventsByUid', () => {
+    return db
+      .securityEvent({
+        ipAddr: '127.0.0.1',
+        name: 'account.create',
+        uid: account.uid,
+      })
+      .then(() => {
+        return db.deleteSecurityEventsByUid({
+          uid: account.uid,
+        });
+      })
+      .then(events => {
+        assert.deepEqual(events, {});
```

Comment:
After this assert, try to get the security events, it should return 0.

Reply:
Nice catch @vbudhram